ssv/proposer.go

```diff
@@ -188,30 +181,20 @@ func (r *ProposerRunner) ProcessPostConsensus(signedMsg *types.SignedPartialSign
 		specSig := phase0.BLSSignature{}
 		copy(specSig[:], sig)
 
-		if r.decidedBlindedBlock() {
-			data, err := r.GetState().DecidedValue.GetBellatrixBlindedBlockData()
+		if r.GetState().DecidedValue.DecidedBlindedBlock() {
```

Comment:
should be in this file not consensus data as it's very specific to this runner

Reply:
we use this func in more than 1 place